README.md

```diff
@@ -1 +1,33 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> The goal of this project is to master all of the tools and best practices I learned about in previous steps. 
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://SoniaKarungi.github.io/Hello-Microverse)
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone this repository to your local machine
```

Comment:
- [ ] Let's add this section also, which contains the necessary code for running the project locally. For ex:-
```
git clone https://github.com/Epaltechs/Hello-World.git
cd Hello-World
```